stdlib/pushbullet/pushbullet.flux

```diff
@@ -1,3 +1,4 @@
+// The Pushbullet package provides functions for sending data to Pushbullet.
```

Comment:
nit, can you use the same capitalization as the package name itself, i.e. `The pushbullet pacakge ...`?